test/integ/aio/test_put_get.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+import filecmp
+import logging
+import os
+from io import BytesIO
+from logging import getLogger
+from os import path
+from unittest import mock
+
+import pytest
+
+from snowflake.connector import OperationalError
+
+try:
+    from snowflake.connector.util_text import random_string
+except ImportError:
+    from test.randomize import random_string
+
+from test.generate_test_files import generate_k_lines_of_n_files
+
+try:
+    from ..parameters import CONNECTION_PARAMETERS_ADMIN
+except ImportError:
+    CONNECTION_PARAMETERS_ADMIN = {}
+
+THIS_DIR = path.dirname(path.realpath(__file__))
+
+logger = getLogger(__name__)
+
+pytestmark = pytest.mark.asyncio
+CLOUD = os.getenv("cloud_provider", "dev")
+
+
+@pytest.mark.skipif(CLOUD not in ["aws", "dev"], reason="only test in aws now")
+@pytest.mark.skipolddriver
```

Comment:
nit: I think you can remove the skipolddriver mark, I configured tox to skip collecting aio tests in old driver test